modules/network/arbitrum.ts

```diff
@@ -68,9 +68,12 @@ export const arbitrumNetworkData: NetworkData = {
         delegationProxy: '0x81cfae226343b24ba12ec6521db2c79e7aeeb310',
     },
     balancer: {
-        vault: '0xba12222222228d8ba445958a75a0704d566bf2c8',
-        swapProtocolFeePercentage: 0.5,
-        yieldProtocolFeePercentage: 0.5,
+        vaultV2Address: '0xba12222222228d8ba445958a75a0704d566bf2c8',
+        v2DefaultSwapFeePercentage: '0.5',
+        v2DefaultYieldFeePercentage: '0.5',
+        vaultV3Address: '0xba12222222228d8ba445958a75a0704d566bf2c8',
+        v3DefaultSwapFeePercentage: '0.5',
+        v3DefaultYieldFeePercentage: '0.5',
```

Comment:
how about namespacing that? eg:
```suggestion
        v2: {
            vault: '0xba12222222228d8ba445958a75a0704d566bf2c8',
            defaultSwapFeePercentage: '0.5',
            defaultYieldFeePercentage: '0.5',
        },
        v3: {
            vault: '0xba12222222228d8ba445958a75a0704d566bf2c8',
            defaultSwapFeePercentage: '0.5',
            defaultYieldFeePercentage: '0.5',
        }
```

Reply:
agree